assets/terraform-modules/aws/flatcar-linux/kubernetes/variables.tf

```
@@ -185,3 +185,9 @@ variable "kube_apiserver_extra_flags" {
   type        = list(string)
   default     = []
 }
+
+variable "encrypt_pods_traffic" {
```

```suggestion
variable "encrypt_pod_traffic" {
```

"Pods traffic" sounds weird to me. I'd say "pod traffic" even when talking about multiple pods.